test/fixtures/3.0/ping/.vscode/settings.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
How do you envision that people will be using this VSCode settings? Do you expect developers to have two instances of VSCode opened, one for the entire connector project and another just for the lb4 app? That does not seem very ergonomic to me.

IMO, we should find project configuration where everything can be edited from a single VSCode opened for the entire connector project.